NitroxModel/Packets/WrapperPacket.cs

```diff
@@ -0,0 +1,16 @@
+namespace NitroxModel.Packets
+{
+    public class WrapperPacket
```

Comment:
Why wrap the packet?

Reply:
Because of LiteNetLib Serializer. See #652 for more information. It was a bit weird.